sdgym/benchmark.py

```diff
@@ -108,7 +113,32 @@ def _create_detailed_results_directory(detailed_results_folder):
         os.makedirs(detailed_results_folder, exist_ok=True)
 
 
-def _setup_output_destination(output_destination, synthesizers, datasets):
+def _setup_output_destination_aws(output_destination, synthesizers, datasets, s3_client):
+    paths = defaultdict(dict)
+    s3_path = output_destination[5:]
```

Comment:
can we make this less mysterious? Maybe make a constant with the prefix and then do `output_destination[len(CONSTANT):]`